Makefile

```diff
@@ -451,3 +451,8 @@ package-rpm-suse package-windows package-standalone-macosx \
 package-generic-unix: $(PACKAGES_SOURCE_DIST_FILE)
 	$(verbose) $(MAKE) -C packaging $@ \
 		SOURCE_DIST_FILE=$(abspath $(PACKAGES_SOURCE_DIST_FILE))
+
+generate-config:
+	rm -rf generated/rabbitmq.*.config
+	./cuttlefish -e . -s ./schema -c rabbitmq.conf.d/rabbitmq.conf -f rabbitmq
```

Comment:
Right, it doesn't build a proper escript out of it right now (rebar escript options are ignored). If you don't care too much about having a "cuttlefish" executable you can have it in `BUILD_DEPS` and call the erlang functions directly through `$(erlang ...)`, otherwise I can fix it in Erlang.mk.